ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/ErrorDialog.kt

```diff
@@ -0,0 +1,35 @@
+package com.revenuecat.purchases.ui.revenuecatui.composables
+
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import com.revenuecat.purchases.ui.revenuecatui.R
+
+@Composable
+internal fun ErrorDialog(
```

Comment:
FWIW, we've started marking all these internal functions with `@JvmSynthetic` for Paywalls V2 and the file itself with `@file:JvmSynthetic`, so they are not available in Java. In case we want to do the same for this and everything else moving forward?